src/buttonsBroker.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { ButtonProps } from './interfaces/buttonProps';
+import {Alert} from 'react-bootstrap';
+
+class ButtonsBroker extends Component<ButtonProps> {
+    constructor(props:any) {
+        super(props);
+    }
+
+    render() {
+      const{ buttonsEnabled, tabItems, deleteSelection, exportSelection, setIntent, closeExportInfo, showAlert, intentEnabled } = this.props;
+
+      let deleteBtn;
+      let exportBtn;
+      let intentBtn;
+      let alertBox;
+
+      if (tabItems.length > 0){
+        if (buttonsEnabled) {
+          deleteBtn = <i id="deleteBtn"
+                         className="fa fa-trash"
+                         title='Delete selected cards'
+                         onClick={() => deleteSelection()} />
+          exportBtn = <i  id="exportBtn" 
+                          className='fa fa-upload' 
+                          title='Export selected visualization into variable'
+                          onClick={(e) => exportSelection()} />
+               
+        } else {
+          deleteBtn = <i id="deleteBtn"
+                         className="fa fa-trash"
+                         style={{opacity: 0.2, cursor: 'not-allowed'}}
+                         title='Select card(s) to delete' />
+          exportBtn = <i  id="exportBtn" 
+                          className= 'fa fa-upload'
+                          style={{opacity: 0.2, cursor: 'not-allowed'}}
+                          title='Select card(s) to export into variable' />
+        }
+
+        if (buttonsEnabled && intentEnabled) {
+          intentBtn = <i  id="intentBtn"
+                          className="fa fa-search"
+                          title='Set selected card as intent'
+                          onClick={() => setIntent()} /> 
+        } else {
+          intentBtn = <i  id="intentBtn"
+                          className="fa fa-search"
+                          style={{opacity: 0.2, cursor: 'not-allowed'}}
+                          title='Select card to set as intent' />
```

Comment:
Maybe you can make this more specific -- "Select 1 card to set as intent". That way you don't need to set a separate warning and it can be under this button.

Reply:
Yeah I think that would be better